app/models/case_contact_report.rb

```diff
@@ -12,12 +12,14 @@ def self.to_csv
   end
 
   def self.report_headers
-    headers = %w[ internal_contact_number duration contact_type contact_made
-                  contact_medium occurred_at added_to_system_at casa_case_number
-                  volunteer_email volunteer_name supervisor_name]
+    headers = %w[internal_contact_number duration]
+
+    headers.concat(CaseContact::CONTACT_TYPES.map { |t| "contact_type: #{t}" })
+
+    headers.concat(%w[ contact_made
```

Comment:
This formatting is weird- one item on a line, then longer lines below. It might be better to make it one item per line if they don't all fit on one line.

Reply:
Good point!